### ISSUE TITLE
Probleme in IAM policy Console Minio

### ISSUE THREAD

**Issue by antony14110** [minio/console#1320]
Hello, 
Users authenticated by OpenID must have access to their bucket (bucket name provided by the "scope" value of the jwt token).
They inherit a policy to see that their bucket but, despite the s3:* action, I do not see the option to download the file in the bucket. Why ?
```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Deny",
            "Action": [
                "s3:DeleteBucket",
                "s3:CreateBucket"
            ],
            "Resource": [
                "arn:aws: s3 ::: *"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:GetBucketLocation",
                "s3:ListAllMyBuckets"
            ],
            "Resource": [
                "arn:aws:s3::: *"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:*"
            ],
            "Resource": [
                "arn:aws:s3:::${jwt: scope} *"
            ]
        }
    ]
}
```
The user can see 
bucket_name
"Created: 2021-12-07T02:35:28Z
Access: R/W"
But when he go to the bucket : He don't see the elements and he can't upload a new document.
The bucket's properties are :
Access Policy: | Public
Replication: | Disabled
Object Locking: | Disabled
Encryption: | Disabled
Versioning:Enable
Quota: Disable
name: console
image: minio/console:v0.12.6

**Comment by vadmeste** [comment]
@antony14110 your policy document doesn't seem to be correct, probably pasting issue, you need to use Github 'Insert Code' feature so your JSON document is properly displayed.

**Comment by antony14110** [comment]
`{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Deny",
            "Action": [
                "s3:CreateBucket",
                "s3:DeleteBucket"
            ],
            "Resource": [
                "arn:aws:s3:::*"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:GetBucketLocation",
                "s3:ListAllMyBuckets"
            ],
            "Resource": [
                "arn:aws:s3:::*"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:*"
            ],
            "Resource": [
                "arn:aws:s3:::${jwt:scope}*"
            ]
        }
    ]
}`

**Comment by antony14110** [comment]
It's better ?

**Comment by vadmeste** [comment]
@antony14110 can you double check the `${jwt:scope}` using jwt.io ? You can get the JWT token using `mc admin trace -v <alias>` and you login in MinIO

**Comment by antony14110** [comment]
Hello,

Thanks for your feedback. I checked with the jwt.io site and got a good result:
"scope": "test1",
Also, the test1 bucket appears in the console, it is the only visible bucket. So I think the ${jwt: scope} tag works. But I can't put a document.

**Comment by echupriyanov** [comment]
I can confirm that the same issue present with LDAP authentication configured.
I have the `testuser` in my LDAP with `test-group` membership.
Also, I've applied the following policy to the `test-group`:
```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "s3:*"
            ],
            "Resource": [
                "arn:aws:s3:::test-*"
            ]
        }
    ]
}

**Comment by echupriyanov** [comment]
UPDATE: all the above applies to manually created users/groups in Minio too

**Comment by dvaldivia** [comment]
we might have fixed this in #1587
